local/e2e/cli-only/e2e_test.go

```diff
@@ -210,6 +210,22 @@ func TestContextMetrics(t *testing.T) {
 			`{"command":"context ls","context":"moby","source":"cli","status":"success"}`,
 		}, usage)
 	})
+
+	t.Run("metrics on build", func(t *testing.T) {
+		s.ResetUsage()
+
+		_ = os.Setenv("BUILDX_CONFIG", "./testdata/buildx-config")
+		c.RunDockerOrExitError("build", "-t", "foo:bar", ".")
+		c.RunDockerOrExitError("buildx", "build", "-t", "foo:bar", ".")
+		c.RunDockerOrExitError("buildx", "--builder", "graviton2", "build", "-t", "foo:bar", ".")
+
+		usage := s.GetUsage()
+		assert.DeepEqual(t, []string{
+			`{"command":"build","context":"moby","source":"cli-docker;buildkit","status":"failure"}`,
+			`{"command":"buildx build","context":"moby","source":"cli-buildx;docker-container","status":"failure"}`,
+			`{"command":"buildx --builder build","context":"moby","source":"cli-buildx;docker-container","status":"failure"}`,
```

Comment:
Nit : why use "-" as a first separator and ";" as a second one in source values here ; it might be more readable (and avoid some possible corner cases in looker) to just use "-" everywhere

Reply:
that's because buildx drivers can be `docker-container`, `docker` or `kubernetes` so I wanted to make a clear distinction of the `cli` and the `builder` being used otherwise we would have smth like `cli-buildx-docker-container` which is not really obvious compared to `cli-buildx;docker-container`. WDYT?

Reply:
right ; that should be ok, I don't foresee any issue with these values in looker